policy/datasets.yaml

```diff
@@ -980,4 +980,43 @@ dcrDiff_kernelSrc:
     persistable:   SourceCatalog
     storage:       FitsCatalogStorage
     tables:        'raw'
-
+makeBrighterFatterKernel_config:
+    description: "Configuration for the brighter-fatter coefficient calculation code."
+    persistable: Config
+    python: lsst.cp.pipe.brighterFatterKernelGeneration.MakeBrighterFatterKernelTaskConfig
+    storage: ConfigStorage
+    tables:
+    - raw
+    - raw_visit
+    template: config/makeBrighterFatterKernel.py
+makeBrighterFatterKernel_metadata:
+    description: "Metadata for the brighter-fatter coefficient calculation code."
+    persistable: PropertySet
+    storage: BoostStorage
+    python: lsst.daf.base.PropertySet
+    tables: raw
+    template: brighterFatter_metadata/brighterFatter_metadata.boost
+brighterFatterKernelNew:
+    description: >
+        New brighter-fatter kernels, such that they do not conflict with brighterFatterKernel in obs_subaru.
+        This will be renamed, and replace the brighterFatterKernel later, but they must exist simultaneously
+        for now.
+    persistable: ignored
+    storage: PickleStorage
+    python: numpy.array
+    tables: raw
+    template: ''
+brighterFatterGain:
+    description: "The per-amplifier gains, as calculated and used by the brighter-fatter task."
+    persistable: ignored
+    storage: PickleStorage
+    python: numpy.array
```

Comment:
Isn't this a dict of amp-or-detector-name (as a str): gain (as a float)? Though as I mentioned in cp_pipe review, if you made it a simple struct-like class then you could store the level as an attribute. Since you are using pickle storage anyway, that does not seem to be much of a hardship.

Reply:
Done.

Reply:
Surely the `python:` entry should be changed to the type of the new custom class?

Reply:
Sorry, I'm confused, but it's also all sorted. I thought that this was for the kernels, not the gains. I made a class for that, and _did_change the python type. This is just a dict, and I have also changed it to not be a lie here. So, unless you dislike that and think that should be a thin class (which I don't agree with, as the gains as by definition at the amp level, so a dict is fine), then this is all nicely done, with the nice idea expressed here applied to the kernels, not the gains.

Does that make sense?

Reply:
Also, I amended the commit to save squashing later, so if you look, look here, or beware the force-push, sorry.

Reply:
The python type says "numpy.array" but apparently it's a dict, so surely it should be "builtins.dict"?

Reply:
Sorry, that's outdated, it does now say `dict`. However, I just did a plain `dict` rather than `buitlins.dict`. Does that matter? (Obviously I don't mind changing it, this is more for my education.)

Reply:
You were, of course, right, I've changed now.

Reply:
I suspect "builtins" is not necessary, but I'm not sure. Of course None may work, in which case I'd go with that.